platform/golden/cfg/debian-11-preseed.cfg

```diff
@@ -0,0 +1,92 @@
+####################
+## General defaults
+
+d-i debian-installer/locale string en_US.UTF-8
+d-i keyboard-configuration/xkb-keymap select us
+d-i pkgsel/install-language-support boolean false
+
+####################
+## Network defaults
+
+d-i netcfg/choose_interface select auto
+d-i netcfg/get_hostname string golden
+d-i netcfg/get_domain string local
+
+
+###################
+## Mirror defaults
+
+d-i mirror/country string manual
+d-i mirror/http/hostname string http.us.debian.org
+d-i mirror/http/directory string /debian
+d-i mirror/http/proxy string
+d-i apt-setup/use_mirror boolean true
+
+#################
+## Time defaults
+
+d-i clock-setup/utc boolean true
+d-i time/zone string UTC
+d-i clock-setup/ntp boolean true
+
+##################
+## Login defaults
+
+d-i passwd/root-login boolean false
+d-i passwd/user-fullname string Artificial Wisdom
+d-i passwd/username string packer
+d-i passwd/user-password password packer
+d-i passwd/user-password-again password packer
+d-i passwd/user-default-groups string users,sudo
+
+###################
+## Select packages
+
+tasksel tasksel/skip-tasks  string standard
+tasksel tasksel/first multiselect openssh-server
+popularity-contest popularity-contest/participate boolean false
+
+d-i pkgsel/include string curl wget openssh-server vim sudo cloud-init
+d-i pkgsel/language-pack-patterns   string
+d-i pkgsel/install-language-support boolean false
+d-i pkgsel/update-policy select unattended-upgrades
+d-i pkgsel/upgrade select full-upgrade
+
+###########
+## Use XFS
+
+d-i partman/default_filesystem string xfs
+d-i partman-auto/method string regular
+
+###################
+## Volume defaults
+
+d-i partman-auto/choose_recipe select atomic
+d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+
+####################
+## Additional media
+d-i apt-setup/cdrom/set-first boolean false
+d-i apt-setup/cdrom/set-next boolean false
+d-i apt-setup/cdrom/set-failed boolean false
+d-i apt-setup/non-free boolean false
+d-i apt-setup/contrib boolean false
+
+#######################
+## Bootloader defaults
+
+d-i grub-installer/only_debian boolean true
+d-i finish-install/reboot_in_progress note
+d-i grub-installer/bootdev  string /dev/sda
+
+#####################
+## Add late commands
+
+d-i preseed/late_command string echo "packer ALL=(ALL:ALL) NOPASSWD:ALL" > /target/etc/sudoers.d/packer && chmod 044 /target/etc/sudoers.d/packer
```

Comment:
what is `/target/etc/`? Should this not be `/etc/`?

Reply:
removed the need for this.  But for completeness:

/target is where the build is mounted during the initial build, so to modify the built filesystem, you have to point at /target/{normal_path}